ui/app/utils/classes/stream-logger.js

```diff
@@ -10,7 +10,7 @@ import classic from 'ember-classic-decorator';
 export default class StreamLogger extends EmberObject.extend(AbstractLogger) {
   reader = null;
 
-  get isSupported() {
+  static get isSupported() {
```

Comment:
question: Should we have a test to document and protect this behavior?

Reply:
Yeah, I mentioned in the PR body that 

> I couldn't find a way to test this since the test suite uses a mock object.

But turns out that's not true 😅 

Added a test now.